pkg/cmd/pipeline/pipeline.go

```diff
@@ -35,6 +35,7 @@ func Command(p cli.Params) *cobra.Command {
 		describeCommand(p),
 		listCommand(p),
 		startCommand(p),
+		logCommand(p),
```

Comment:
could we maintain the alphabetical order? as we are doing it at other places as well